src/controllers/carpools.ts

```diff
@@ -0,0 +1,194 @@
+/// <reference path="../../typings/express-session/express-session.d.ts" />
```

Comment:
I was going to merge the user controller methods into 1 file. I guess you can just leave it as is for now and we can merge the carpool methods when we merge the user controller methods.